Entity/NotificationMessageInterface.php

```diff
@@ -21,9 +20,19 @@ public function getRecipientsItems(): Collection;
     public function getMessage(): ? string;
 
     /**
+     * @param null|string $message
+     */
+    public function setMessage(?string $message): void;
```

Comment:
can message be nullable? What is the usecase then?

Reply:
Just title for the push notification.